data/test/scenarios/macro_tests/test_force_chance_to_hit_macro.cfg

```diff
@@ -99,20 +109,13 @@
                 [/destination]
             [/attack]
         [/do_command]
-        [store_unit]
-            [filter]
-                id=alice
-            [/filter]
-            variable=a
-        [/store_unit]
-        [store_unit]
-            [filter]
-                id=bob
-            [/filter]
-            variable=b
-        [/store_unit]
-        {ASSERT ({VARIABLE_CONDITIONAL a.hitpoints equals 100})}
-        {ASSERT ({VARIABLE_CONDITIONAL b.hitpoints equals 100})}
+
         {SUCCEED}
     [/event]
+
+    [event]
+        name=attacker hits,defender hits
+        first_time_only=no
+        {FAIL}

```

Comment:
Nice redesign of the test.

This file also has DOS-style line endings, might be a good time to convert it.

Reply:
Updated.